File: py/Chapter6/ps1.py
a = int(input("Enter 1st no.: "))
b = int(input("Enter 2nd no.: "))
c = int(input("Enter 3rd no.: "))
d = int(input("Enter 4th no.: "))

print(a,b,c,d)

if(a>b and a>c and a>d):
    print("1st no. is greater")

elif(b>a and b>c and b>d):
    print("2nd no. is greater")

elif(c>b and c>a and c>d):
    print("3rd no. is greater")

elif(d>b and d>c and d>a):
    print("4th no. is greater")

else:
    print("invalid")

File: Lab-Assingments/C/ass2/ps1.c
#include<stdio.h>
int main(){
    int choice;
    float a, b, res;
    printf("Calculation\n");
    printf("1.Addition\n");
    printf("2.Substraction\n");
    printf("3.Multiplication\n");
    printf("4.Division\n");
    printf("Enter your choice:\n ");
    scanf("%d",&choice);
    printf("Enter two number of your choice: ");
    scanf("%f %f",&a,&b);
    switch (choice)
    {
    case 1:
        res = a+b;
        printf("%f",res);
        break;
    case 2:
        res = a-b;
        printf("%f",res);
        break;
    case 3:
        res = a*b;
        printf("%f",res);
        break;
    case 4:
        if(b!=0){
        res = a/b;
        printf("%f",res);
        }
        else{
            printf("Number cannot be divisible by zero.");
        }
        break;

    default:
        printf("Invalid");
        break;
    }
}
    